### ISSUE TITLE
Stop Using <bits/stdc++.h> for Portability

### ISSUE THREAD

**Issue by arnavb** [OpenGenus/cosmos#1915]
**This is a(n):**

- [ ] New algorithm
- [ ] Update to an existing algorithm
- [x] Error
- [x] Proposal to the Repository

**Details:**

I was googling up about the header file `<bits/stdc++.h>`, and found that it is consider generally bad practice to use it, because it is nonportable (It only works on g++, not MSVC or XCode).

1. ([Source](https://stackoverflow.com/a/31816096/6525260))
2. ([Source](https://stackoverflow.com/a/33764128/6525260))

Thus, I vote not using this header file and instead directly including files from the standard.

**Comment by tbhaxor** [comment]
@arnavb yes its a bad practice coz it binds all header files and std library

**Comment by arnavb** [comment]
I will open a PR soon that will attempt to solve this. Also, I think we should mandate any new PRs to NOT use the header file, but to directly include the STL files.
